Update documentation to clarify the default format for logs (#4953)
containous/traefik: docs/configuration/logs.md

[traefikLog]
  filePath = "/path/to/traefik.log"

>   format   = "json" # Default: "common"

Reviewer: ```suggestion
  format   = "json"
```

[accessLog]
  filePath = "/path/to/access.log"

>   format = "json" # Default: "common"

Reviewer: ```suggestion
  format = "json"
```

Author: Isn't this true for the `accessLog` ?

For more information about the CLI, see the documentation about [Traefik command](/basics/#traefik).

```bash
--logLevel="DEBUG"
--traefikLog.filePath="/path/to/traefik.log"

> --traefikLog.format="json"  # Default: "common"

Reviewer: ```suggestion
--traefikLog.format="json"
```

--accessLog.filePath="/path/to/access.log"

> --accessLog.format="json"  # Default: "common"

Reviewer: ```suggestion
--accessLog.format="json"
```

Author: Isn't this true for the `accessLog` ?

  filePath = "/path/to/traefik.log"
```

> To switch to JSON format instead of [Common Log Format (CLF)](#clf-common-log-format), specify `json` as the format:

Reviewer: ```suggestion
To switch to JSON format instead of standard format (`common`), specify `json` as the format:
```

```toml
[traefikLog]
  filePath = "/path/to/traefik.log"

>   format   = "json" # Default: "common"

Reviewer: ```suggestion
  format   = "json"
```
